status/pricing.py

```diff
@@ -226,8 +227,15 @@ def _calculate_product_price(self, product, all_component_prices):
             component_price_d = all_component_prices[str(reagent)]
             price += component_price_d['price_per_unit_in_sek']
 
-        external_price = price + price * product['Re-run fee']
-        return price, external_price
+        price_academic = price + price * product['Re-run fee']
+
+        full_cost_fee = product.get('Full cost fee', '')
+        if full_cost_fee == '':
+            full_cost_fee = '0.0'
```

Comment:
Could this be simplified to `product.get('Full cost fee', '0.0')`?

Reply:
I thought about it, but the case I want to catch as well is when the empty string is actually what's found in product. This was the solution I came up with. I guess it could be better to remove the empty strings from the databases though...

Reply:
Ah, I see, that makes sense! 👍